data/presets/presets/amenity/prison.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icon": "prison",
+    "fields": [
+        "operator",
+        "access"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "prison",
```

Comment:
Don't need to add "prison" as a search term if it is the tag value - it's already included.  The other terms you added are 👍 ..